src/styles/block/main.scss

```diff
@@ -0,0 +1,58 @@
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $cellSize * 7 + 10;
+  margin-left: 4px;
+  padding-inline: 10px;
+  gap: $cellValue;
```

Comment:
The variable '$cellValue' used in 'gap' might not be defined. Ensure that '$cellValue' is declared and assigned a value in your variables file.